tests/view/items/test_items.py

```diff
@@ -29,7 +29,7 @@ def test_list(client):
 
 def test_create(client):
     """ Test create is okay """
-    assert client.get("/items/create").status_code == HTTPStatus.OK
+    assert client.post("/items/create").status_code == HTTPStatus.OK
```

Comment:
@paulodamaso It's interesting, I thought here will be `http 302`. Let's test it.